model.go

```diff
@@ -154,6 +154,15 @@ type Model interface {
 	PasswordHash() string
 
 	AddBlockDevice(string, BlockDeviceArgs) error
+
+	// SetEnvironVersion sets the environment version to the specified value.
+	//
+	// The version must be a non-negative integer. The caller is responsible
+	// for ensuring the version is valid and compatible with dependent systems.
+	//
+	// Updating the environment version will affect subsequent operations that
```

Comment:
Don't think we need this line either :). This seems like a given to me.

Reply:
Yup haha but I wasn't able to think of another post side effect for this. I'll remove it then thanks Tom :)